packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,105 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const ignoredOperations = [
+  /* missing route */
+  /* #5694 */
+  "getSyncCommitteeRewards",
+  "getBlockRewards",
+  "getAttestationsRewards",
+  "getDepositSnapshot", // Won't fix for now, see https://github.com/ChainSafe/lodestar/issues/5697
+  "getBlindedBlock", // #5699
+  "getNextWithdrawals", // #5696
+  "getDebugForkChoice", // #5700
+  /* #6080 */
+  "getLightClientBootstrap",
+  "getLightClientUpdatesByRange",
+  "getLightClientFinalityUpdate",
+  "getLightClientOptimisticUpdate",
+  "getPoolBLSToExecutionChanges",
+  "submitPoolBLSToExecutionChange",
+];
+
+const ignoredProperties: Record<string, IgnoredProperty> = {
+  /*
+   #5693
+   missing finalized
+   */
+  getStateRoot: {response: ["finalized"]},
+  getStateFork: {response: ["finalized"]},
+  getStateFinalityCheckpoints: {response: ["finalized"]},
+  getStateValidators: {response: ["finalized"]},
+  getStateValidator: {response: ["finalized"]},
+  getStateValidatorBalances: {response: ["finalized"]},
+  getEpochCommittees: {response: ["finalized"]},
+  getEpochSyncCommittees: {response: ["finalized"]},
+  getStateRandao: {response: ["finalized"]},
+  getBlockHeaders: {response: ["finalized"]},
+  getBlockHeader: {response: ["finalized"]},
+  getBlockV2: {response: ["finalized"]},
+  getBlockRoot: {response: ["finalized"]},
+  getBlockAttestations: {response: ["finalized"]},
+  getStateV2: {response: ["finalized"]},
+
+  /* 
+   #6168
+   /query/syncing_status - must be integer
+   */
+  getHealth: {request: ["query.syncing_status"]},
+
+  /**
+   * #6185
+   *  - must have required property 'query'
+   */
+  getBlobSidecars: {request: ["query"]},
+
+  /* 
+   #4638 
```

Comment:
It's your choice, but I prefer to have a complete link eg. `https://github.com/ChainSafe/lodestar/issues/4638` in comments, easier to identify if we are pointing to PR or issue and also easy to navigate with one click.